ush/ufsda/misc_utils.py

```diff
@@ -10,3 +15,38 @@ def isTrue(str_in):
     else:
         status = False
     return status
+
+
+def create_batch_job(job_config, working_dir, exe_path, yaml_path):
+    """
+    create a batch job submission shell script
+    """
+    # open up a file for writing
+    batch_script = os.path.join(working_dir, 'submit_job.sh')
+    with open(batch_script, 'w') as f:
+        f.write('#!/bin/bash\n')
+        if job_config['machine'] in ['orion', 'hera']:
```

Comment:
I would suggest using `scheduler` instead of `machine` where these kind of decisions need to be made.

Reply:
Yeah I thought about that, but wanted to make it simpler for the user. Should I do the workflow approach where scheduler is the if statement but it is determined based on a dict (`if scheduler[job_config['machine']] == 'slurm'`)?

Reply:
@aerorahul how is this?